hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/LedgerConfig.java

```diff
@@ -28,7 +29,7 @@ public record LedgerConfig(
         @ConfigProperty(defaultValue = "5000") int maxAutoAssociations,
         @ConfigProperty(defaultValue = "100") int numSystemAccounts,
         @ConfigProperty(defaultValue = "5000000000000000000") long totalTinyBarFloat,
-        @ConfigProperty(defaultValue = "0x03") String id,
+        @ConfigProperty(defaultValue = "0x03") Bytes id,
```

Comment:
Shouldn't default value change ?

Reply:
default values are converted the same as the properties are.

Reply:
I see. Could you please point to the place where conversion is happening.

Reply:
I see `BytesConverter`. Never mind. Thanks

Reply:
It is used when the type (Bytes) is detected.  It's registered on both the config builder and the test config builder in their constructors, and the config provider searches for all the annotated config data classes on startup, matches each attribute to a registered converter, and then fills in default values using the converter found before reading the config sources to find overrides.
If it doesn't find a converter, then it throws an exception during startup (which is why the workflow classes had to be fixed in this PR, because they were using the platform test builder, which doesn't have the Bytes converter).